.cicd/layer_template.yaml

```diff
@@ -0,0 +1,19 @@
+﻿AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Serverless Transform required for Type AWS::Serverless::LayerVersion for resource LambdaLayer